tests/cli/output/help.txt

```diff
@@ -69,6 +69,6 @@ Note: Use comments containing ' *INDENT-OFF*' and ' *INDENT-ON*' to disable
       processing of parts of the source file (these can be overridden with
       enable_processing_cmt and disable_processing_cmt).
 
-There are currently 645 options and minimal documentation.
+There are currently 646 options and minimal documentation.
```

Comment:
Hmm this is unusual, I have the same change and had to manually remove it in my PR as I did not introduce a new option. This PR also does not add a new option. Where is this change coming from?

Reply:
I was noticing unexpected CI errors on #1821 also... it's because master is currently broken. (Possibly two PR's that added options both got merged; if they didn't add them in similar places, the PR's might have both passed CI, but ended up with the wrong total number of options.)

I would encourage a separate PR to fix that, or (if it's just the option count that's off) even someone with permission to just fix it and push straight to master.

Reply:
@gmaurel  Ok I can't do that the master has a lock.